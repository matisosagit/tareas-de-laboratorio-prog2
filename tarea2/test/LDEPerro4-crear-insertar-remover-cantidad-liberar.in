crearLDEPerrosVacia

# Creo e inserto el perro 1
crearFecha 08/09/2021
crearPerro 1 Volta 4 10 : Labrador Retriever, energético y juguetón, pelaje marrón claro
insertarLDEPerros

# Consulto la cantidad
cantidadLDEPerros

# Creo e inserto el perro 2
crearFecha 22/10/2022
crearPerro 2 Titan 8 7 Un Chihuahua valiente y protector con una gran personalidad en un cuerpo pequeño
insertarLDEPerros

# Consulto la cantidad
cantidadLDEPerros

# Creo e inserto el perro 3
crearFecha 14/07/2018
crearPerro 3 Ampere 7 5 Un Boxer curioso y determinado, siempre explorando y metiéndose en pequeñas travesuras
insertarLDEPerros

# Consulto la cantidad
cantidadLDEPerros

# Creo e inserto el perro 4
crearFecha 01/02/2008
crearPerro 4 Turing 3 9 Un Bulldog Inglés tranquilo y cariñoso que disfruta de largas siestas y abrazos
insertarLDEPerros

# Consulto la cantidad
cantidadLDEPerros

removerPerroLDEPerros 3
imprimirLDEPerros

# Consulto la cantidad
cantidadLDEPerros

removerPerroLDEPerros 4
imprimirLDEPerros

# Consulto la cantidad
cantidadLDEPerros

removerPerroLDEPerros 2
imprimirLDEPerros

# Consulto la cantidad
cantidadLDEPerros

# Creo e inserto el perro 5
crearFecha 19/11/2001
crearPerro 5 Turing 2 9 Casi marron, cachorro grande 
insertarLDEPerros

# Consulto la cantidad
cantidadLDEPerros

removerPerroLDEPerros 5

# Consulto la cantidad
cantidadLDEPerros

removerPerroLDEPerros 1

# Consulto la cantidad
cantidadLDEPerros

liberarLDEPerros
Fin
