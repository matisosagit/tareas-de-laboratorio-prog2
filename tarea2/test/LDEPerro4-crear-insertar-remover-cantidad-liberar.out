1>2>#  Creo e inserto el perro 1.
3>4>5>6>#  Consulto la cantidad.
7>La cantidad de perros en la lista es 1
8>#  Creo e inserto el perro 2.
9>10>11>12>#  Consulto la cantidad.
13>La cantidad de perros en la lista es 2
14>#  Creo e inserto el perro 3.
15>16>17>18>#  Consulto la cantidad.
19>La cantidad de perros en la lista es 3
20>#  Creo e inserto el perro 4.
21>22>23>24>#  Consulto la cantidad.
25>La cantidad de perros en la lista es 4
26>Perro removido de la lista:
Perro 3
Nombre: Ampere
Edad: 7
Descripcion:  Un Boxer curioso y determinado, siempre explorando y metiéndose en pequeñas travesuras
Fecha de ingreso: 14/7/2018
Vitalidad: 5
27>LDE Perros:
Perro 4
Nombre: Turing
Edad: 3
Descripcion:  Un Bulldog Inglés tranquilo y cariñoso que disfruta de largas siestas y abrazos
Fecha de ingreso: 1/2/2008
Vitalidad: 9
Perro 1
Nombre: Volta
Edad: 4
Descripcion:  : Labrador Retriever, energético y juguetón, pelaje marrón claro
Fecha de ingreso: 8/9/2021
Vitalidad: 10
Perro 2
Nombre: Titan
Edad: 8
Descripcion:  Un Chihuahua valiente y protector con una gran personalidad en un cuerpo pequeño
Fecha de ingreso: 22/10/2022
Vitalidad: 7
28>#  Consulto la cantidad.
29>La cantidad de perros en la lista es 3
30>Perro removido de la lista:
Perro 4
Nombre: Turing
Edad: 3
Descripcion:  Un Bulldog Inglés tranquilo y cariñoso que disfruta de largas siestas y abrazos
Fecha de ingreso: 1/2/2008
Vitalidad: 9
31>LDE Perros:
Perro 1
Nombre: Volta
Edad: 4
Descripcion:  : Labrador Retriever, energético y juguetón, pelaje marrón claro
Fecha de ingreso: 8/9/2021
Vitalidad: 10
Perro 2
Nombre: Titan
Edad: 8
Descripcion:  Un Chihuahua valiente y protector con una gran personalidad en un cuerpo pequeño
Fecha de ingreso: 22/10/2022
Vitalidad: 7
32>#  Consulto la cantidad.
33>La cantidad de perros en la lista es 2
34>Perro removido de la lista:
Perro 2
Nombre: Titan
Edad: 8
Descripcion:  Un Chihuahua valiente y protector con una gran personalidad en un cuerpo pequeño
Fecha de ingreso: 22/10/2022
Vitalidad: 7
35>LDE Perros:
Perro 1
Nombre: Volta
Edad: 4
Descripcion:  : Labrador Retriever, energético y juguetón, pelaje marrón claro
Fecha de ingreso: 8/9/2021
Vitalidad: 10
36>#  Consulto la cantidad.
37>La cantidad de perros en la lista es 1
38>#  Creo e inserto el perro 5.
39>40>41>42>#  Consulto la cantidad.
43>La cantidad de perros en la lista es 2
44>Perro removido de la lista:
Perro 5
Nombre: Turing
Edad: 2
Descripcion:  Casi marron, cachorro grande 
Fecha de ingreso: 19/11/2001
Vitalidad: 9
45>#  Consulto la cantidad.
46>La cantidad de perros en la lista es 1
47>Perro removido de la lista:
Perro 1
Nombre: Volta
Edad: 4
Descripcion:  : Labrador Retriever, energético y juguetón, pelaje marrón claro
Fecha de ingreso: 8/9/2021
Vitalidad: 10
48>#  Consulto la cantidad.
49>La cantidad de perros en la lista es 0
50>51>Fin.
